ui/src/main.rs

```diff
@@ -359,16 +363,20 @@ pub fn app_bootstrap(cx: Scope) -> Element {
     let mut state = if STATIC_ARGS.use_mock {
```

Comment:
use `State::load()` instead of this `if` statement. looks like a merge conflict caused a reversion.